### PR TITLE
More tests & remove Travis CI

### PR DESCRIPTION
**Pull request by andreeaflorescu**
Added a buildkite pipeline for testing the fam-wrappers features and other combination of features that are not tested with rust-vmm-ci.

Removed travis configuration as we are using Buildkite now.

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    plugins:
+      - docker#v3.0.1:
+          image: "rustvmm/dev:v4"
+          always-pull: true
+
+  # Test release build with kernel bindings 4.20 and fam-wrappers feature.
+  - label: "build-v4.20-x86"
+    commands:
+     - cargo build --release --features=fam-wrappers,kvm-v4_20_0
+     - cargo build --release --features=fam-wrappers,kvm-v4_14_0 --target x86_64-unknown-linux-musl
```
Comment (sameo) [inline .buildkite/pipeline.yml]: I think you meant:
```
--features=fam-wrappers,kvm-v4_20_0
```

Comment (andreeaflorescu) [reply .buildkite/pipeline.yml]: I should actually remove this as kvm-v4_20_0+fam_wrappers is already tested.

Comment (sameo) [reply .buildkite/pipeline.yml]: as `all-features` ?

Comment (andreeaflorescu) [reply .buildkite/pipeline.yml]: Yap.

Comment (andreeaflorescu) [reply .buildkite/pipeline.yml]: Ah. No, you're right. We don't test it with musl

Comment (andreeaflorescu) [reply .buildkite/pipeline.yml]: @sameo can you pls take another look?

**Pull request merged**